metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -765,6 +765,14 @@
                 <type>string</type>
                 <mandatory-flag>false</mandatory-flag>
             </attribute-definition>
+            <attribute-definition attribute-id="stickyioProductSwapEnabled">
+                <display-name xml:lang="x-default">Product Swap Enabled</display-name>
```

Comment:
I think that it's better to display "Enable Product Swap", it's easier to understand and is more consistent with the existing label 'Enable sticky.io'...